src/medals/core.cljs

```diff
@@ -0,0 +1,26 @@
+(ns medals.core
+  (:require
+   [reagent.dom :as rdom]
+   [re-frame.core :as re-frame]
+   [medals.events :as events]
+   [day8.re-frame.http-fx]
+   [medals.views :as views]
+   [medals.config :as config]))
+
+
+(defn dev-setup []
```

Comment:
Oh thats interesting creature :D Why you need this? If it is not used please remove it ;)